buildsystem/python.cmake

```diff
@@ -406,57 +424,70 @@ function(python_finalize)
 		endif()
 	endforeach()
 
+	##################################################
+	# code generation
+
+	codegen_run()
```

Comment:
- The `add_subdirectory(libopenage)` is before `pyhon_finalize()` in the top-level `CMakeLists.txt`.
- `codegen_run` implicitly depends on `execute_process([compilepy.py ...])` here to link/copy the python modules (including `openage.codegen`) to the binary directory.

An appropriate fix would be to move `openage.codegen` to `buildsystem` and invoke codegen from the source directory.

Reply:
The `codegen` module is part of the `openage` module because it uses code from `openage.convert` in order to generate code. We can't move `openage.codegen` to `buildsystem` easily because of that. We'll leave it as-is for now.